src/transformers/models/xglm/modeling_xglm.py

```diff
@@ -75,11 +75,34 @@
             - 0 for tokens that are **masked**.
 
             [What are attention masks?](../glossary#attention-mask)
-        head_mask (`torch.Tensor` of shape `(encoder_layers, encoder_attention_heads)`, *optional*):
-            Mask to nullify selected heads of the attention modules in the encoder. Mask values selected in `[0, 1]`:
+        position_ids (`torch.LongTensor` of shape `(batch_size, sequence_length)`, *optional*):
```

Comment:
`XGLM_INPUTS_DOCSTRING` did not contain the actual arguments of `forward`, and `forward` had a correct docstring (which meant that the docs had a two sets of inputs 👀 )

This PR fixes that